tsconfig.base.json

```diff
@@ -10,7 +10,7 @@
     "experimentalDecorators": true,
     "noImplicitAny": true,
     "importHelpers": false,
-    "target": "es5",
+    "target": "es2015",
```

Comment:
why this target when `es2019` is targeted in the lib below.

Reply:
Can we keep these targets consistent? also can we make it newer then this?

Reply:
they're different because the `lib` setting defines what type definitions are loaded, while this defines how the actual transpilation to plain JS is done. We can potentially target a newer version but we're still trying to support fairly old NodeJS versions. According to this table we could probably get away with ES2019?
https://node.green/